tests/commands/update/test_update_case_cmd.py

```diff
@@ -112,6 +113,37 @@ def test_update_case_vcf_path(mock_app, case_obj, vcf_key, custom_temp_file):
     assert res["vcf_files"][vcf_key] == vcf_path
 
 
+@pytest.mark.parametrize(
+    "omics_key",
+    [
+        "fraser",
+        "outrider",
+    ],
+)
+def test_update_case_vcf_path(mock_app, case_obj, omics_key, custom_temp_file):
+    """Test the CLI function that updates the case document with paths to different variant types."""
+
+    # GIVEN a new VCF variant path to save in case document
+    omics_path = str(custom_temp_file(".tsv"))
+
+    ## GIVEN a CLI object
+    runner = mock_app.test_cli_runner()
+    ## WHEN updating the VCF path of a specific variant type
+    file_type_option = "--"
+    file_type_option += omics_key.replace("_", "-")
```

Comment:
Is this needed? Aren't omics keys `fraser` and `outrider`?

Reply:
So far yes! I just kept it for symmetry and future proofing as a general rule. But it is not used now. 🤷‍♂️